bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/bno/GetMembershipsFlowResponder.kt

```diff
@@ -22,8 +22,6 @@ class GetMembershipListFlowResponder(flowSession : FlowSession) : BusinessNetwor
         //build memberships snapshot
         flowSession.receive<MembershipListRequest>()
         val databaseService = serviceHub.cordaService(DatabaseService::class.java)
-        val memberships = databaseService.getActiveMemberships()
-
-        flowSession.send(MembershipsListResponse(memberships))
+        flowSession.send(MembershipsListResponse(databaseService.getAllMemberships()))
```

Comment:
This leads me to think that although we now have a member being able to be in multiple networks we cannot have a BNO running more than one network. Which admittedly is not subject of this PR.